Gateway/Startup.cs

```diff
@@ -23,22 +23,20 @@ public class Startup : BaseApiInfo
   {
     public const string CorsPolicyName = "LtDoCorsPolicy";
 
-    private readonly BaseServiceInfoConfig _serviceInfoConfig;
-
     private IConfiguration Configuration { get; }
 
     public Startup(IConfiguration configuration)
     {
       Configuration = configuration;
 
-      _serviceInfoConfig = Configuration
+      BaseServiceInfoConfig serviceInfoConfig = Configuration

```

Comment:
why you changed serviceInfoConfig from private readonly?

Reply:
It was used just like local variable